torchrl/modules/tensordict_module/probabilistic.py

```diff
@@ -50,11 +50,12 @@ class ProbabilisticTensorDictModule(TensorDictModule):
         module (nn.Module): a nn.Module used to map the input to the output parameter space. Can be a functional
             module (FunctionalModule or FunctionalModuleWithBuffers), in which case the `forward` method will expect
             the params (and possibly) buffers keyword arguments.
-        dist_param_keys (str or iterable of str): key(s) that will be produced
+        dist_param_keys (str or iterable of str or dict): key(s) that will be produced
```

Comment:
I know we don't enforce type hinting but in this case i think it's worth amending the type hint below